Compiling template w/o 3d and bullet

<!-- Please search existing issues for potential duplicates before filing yours:
https://github.com/godotengine/godot/issues?q=is%3Aissue
-->

**Godot version:**
<!-- Specify commit hash if non-official. -->
Godot 3.1.dev (eeab3502d)

**OS/device including version:**
<!-- Specify GPU model and drivers if graphics-related. -->
Macosx 10.13.4

**Issue description:**
<!-- What happened, and what was expected. -->
I'm trying to compile the minimal godot 2d template for android. I disabled many modules. My actual building command is: 
```
scons platform=android target=release android_neon=no deprecated=no xml=no disable_3d=yes android_stl=yes game_center=no store_kit=no icloud=no module_bmp_enabled=no module_bullet_enabled=no module_dds_enabled=no module_mobile_vr_enabled=no module_opus_enabled=no module_pvr_enabled=no module_svg_enabled=no module_tga_enabled=no module_theora_enabled=no module_visual_script_enabled=no module_webm_enabled=no verbose=no ndk_platform=android-24 -j12
```
And it fails with error:
```
[ 83%] progress_finish(["progress_finish"], [])
[ 91%] Linking Shared Library ==> bin/libgodot.android.opt.armv7.so
[ 91%] clang++: warning: argument unused during compilation: '-dynamiclib'
scene/libscene.android.opt.armv7.a(skeleton.android.opt.armv7.o):scene/3d/skeleton.cpp:function _pb_stop_simulation(Node*): error: undefined reference to 'PhysicalBone::set_simulate_physics(bool)'
scene/libscene.android.opt.armv7.a(skeleton.android.opt.armv7.o):scene/3d/skeleton.cpp:function _pb_stop_simulation(Node*): error: undefined reference to 'PhysicalBone::set_static_body(bool)'
scene/libscene.android.opt.armv7.a(skeleton.android.opt.armv7.o):scene/3d/skeleton.cpp:function _pb_stop_simulation(Node*): error: undefined reference to 'typeinfo for PhysicalBone'
scene/libscene.android.opt.armv7.a(skeleton.android.opt.armv7.o):scene/3d/skeleton.cpp:function _pb_start_simulation(Skeleton const*, Node*, Vector<int> const&): error: undefined reference to 'PhysicalBone::set_simulate_physics(bool)'
scene/libscene.android.opt.armv7.a(skeleton.android.opt.armv7.o):scene/3d/skeleton.cpp:function _pb_start_simulation(Skeleton const*, Node*, Vector<int> const&): error: undefined reference to 'PhysicalBone::set_simulate_physics(bool)'
scene/libscene.android.opt.armv7.a(skeleton.android.opt.armv7.o):scene/3d/skeleton.cpp:function _pb_start_simulation(Skeleton const*, Node*, Vector<int> const&): error: undefined reference to 'PhysicalBone::set_static_body(bool)'
scene/libscene.android.opt.armv7.a(skeleton.android.opt.armv7.o):scene/3d/skeleton.cpp:function _pb_start_simulation(Skeleton const*, Node*, Vector<int> const&): error: undefined reference to 'typeinfo for PhysicalBone'
scene/libscene.android.opt.armv7.a(skeleton.android.opt.armv7.o):scene/3d/skeleton.cpp:function _physical_bones_add_remove_collision_exception(bool, Node*, RID): error: undefined reference to 'typeinfo for CollisionObject'
clang++: error: linker command failed with exit code 1 (use -v to see invocation)
scons: *** [bin/libgodot.android.opt.armv7.so] Error 1
scons: building terminated because of errors.
```
I think that 3d/skeleton.cpp should be disabled by flag `disable_3d=yes`. It wants to PhysicalBody and when I enable bullet module it requires enabling full 3d support.
So I can not build android template without any 3d and bullet physics.

**Steps to reproduce:**


**Minimal reproduction project:**
<!-- Recommended as it greatly speeds up debugging. Drag and drop a zip archive to upload it. -->

`scons platform=android target=release disable_3d=yes module_bullet_enabled=no verbose=no ndk_platform=android-24 -j12`

```
[ 11%] Compiling ==> modules/gdnative/gdnative/string_name.cpp
In file included from modules/csg/register_types.cpp:33:
In file included from modules/csg/csg_gizmos.h:36:
In file included from ./editor/spatial_editor_gizmos.h:34:
In file included from ./editor/plugins/spatial_editor_plugin.h:34:
In file included from ./editor/editor_node.h:56:
In file included from ./editor/inspector_dock.h:34:
In file included from ./editor/animation_track_editor.h:18:
In file included from ./scene/animation/animation_cache.h:34:
./scene/3d/skeleton.h:83:4: error: use of undeclared identifier 'physical_bone'
                        physical_bone = NULL;
                        ^
./scene/3d/skeleton.h:84:4: error: use of undeclared identifier 'cache_parent_physical_bone'
                        cache_parent_physical_bone = NULL;
                        ^
In file included from modules/csg/csg_gizmos.cpp:31:
In file included from modules/csg/csg_gizmos.h:36:
In file included from ./editor/spatial_editor_gizmos.h:34:
In file included from ./editor/plugins/spatial_editor_plugin.h:34:
In file included from ./editor/editor_node.h:56:
In file included from ./editor/inspector_dock.h:34:
In file included from ./editor/animation_track_editor.h:18:
In file included from ./scene/animation/animation_cache.h:34:
./scene/3d/skeleton.h:83:4: error: use of undeclared identifier 'physical_bone'
                        physical_bone = NULL;
                        ^
[ 11%] Compiling ==> modules/gdnative/gdnative/basis.cpp
./scene/3d/skeleton.h:84:4: error: use of undeclared identifier 'cache_parent_physical_bone'
                        cache_parent_physical_bone = NULL;
                        ^
[ 12%] Compiling ==> modules/gdnative/gdnative/gdnative.cpp
[ 12%] Compiling ==> modules/gdnative/gdnative/vector3.cpp
[ 12%] Compiling ==> modules/gdnative/gdnative/rid.cpp
[ 12%] Compiling ==> modules/gdnative/gdnative/vector2.cpp
[ 12%] Compiling ==> modules/gdnative/gdnative/quat.cpp
2 errors generated.
scons: *** [modules/csg/register_types.android.opt.armv7.neon.o] Error 1
2 errors generated.
scons: *** [modules/csg/csg_gizmos.android.opt.armv7.neon.o] Error 1
scons: building terminated because of errors.
```

--- a/scene/3d/skeleton.h
+++ b/scene/3d/skeleton.h
@@ -80,8 +80,10 @@ class Skeleton : public Spatial {
 			ignore_animation = false;
 			custom_pose_enable = false;
 			disable_rest = false;
+#ifndef _3D_DISABLED
 			physical_bone = NULL;
 			cache_parent_physical_bone = NULL;
+#endif // _3D_DISABLED
 		}
 	};
 

